### DISCUSSION TITLE
Bootstrap Studio

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1185] [Icon request] [answered]
Bootstrap Studio: [https://bootstrapstudio.io](https://bootstrapstudio.io)
![iu](https://user-images.githubusercontent.com/32424458/120096843-a73c4000-c0fb-11eb-8bc0-c97b6328b882.png)

Thank you

**Comment by blaichch** [comment, marked as answer]
Here you go. Hope you like it.
<img width="512" alt="Bootstrap Studio" src="https://user-images.githubusercontent.com/12624897/120103701-30319680-c151-11eb-9fc1-70a74430151a.png">

**Reply by ghost** [reply]
Holy cow that was fast. Thank you, that looks perfect.
